pkg/intermediate/aggregate_test.go

```diff
@@ -169,8 +173,10 @@ func createDataMsgForSrc(t *testing.T, isIPv6 bool, isIntraNode bool, isUpdatedR
 	ie11 = entities.NewInfoElementWithValue(entities.NewInfoElement("flowType", 137, 1, registry.AntreaEnterpriseID, 1), antreaFlowType)
 	tmpElement, _ = registry.GetInfoElement("flowEndReason", registry.IANAEnterpriseID)
 	ie12 := entities.NewInfoElementWithValue(tmpElement, flowEndReason)
+	tmpElement, _ = registry.GetInfoElement("tcpState", registry.AntreaEnterpriseID)
+	ie13 := entities.NewInfoElementWithValue(tmpElement, tcpState)
 
-	elements = append(elements, ie1, ie2, ie3, ie4, ie5, ie6, ie7, ie8, ie9, ie10, ie11, ie12)
+	elements = append(elements, ie1, ie2, ie3, ie4, ie5, ie6, ie7, ie8, ie9, ie10, ie11, ie12, ie13)
```

Comment:
I think we need to improve this test case input to make it easily extendible. We can take it up in the future. Maybe creating an issue is better.